src/rust/engine/process_execution/src/local.rs

```diff
@@ -850,7 +850,11 @@ pub fn setup_run_sh_script(
     let formatted_assignment = format!("{key}={arg_str}");
     env_var_strings.push(formatted_assignment);
   }
-  let stringified_env_vars: String = env_var_strings.join(" ");
+  let stringified_env_var_export: String = if env_var_strings.is_empty() {
+    "".to_owned()
+  } else {
+    "export ".to_owned() + &env_var_strings.join(" ")
```

Comment:
format!("export {}", &env_var_strings.join(" ")) would be more idiomatic I think - I'm slightly surprised clippy doesn't bark.

Reply:
clippy seemed happy with this, but I will change.